PR 4268 introduced build failures for projects with text logging disabled

| | |
|:---|:---|
|**_F´ Version_**|  devel cc2522b257d81d4c7f36bc33e829122f4165f659|
|**_Affected Component_**|   Build|
---
## Problem Description

The updates to always generate JSON in 
[Add fpp-to-json to FPP model sub-build (](https://github.com/nasa/fprime/commit/cc2522b257d81d4c7f36bc33e829122f4165f659) cause the build to fail if text logging is disabled. @thomas-bc  was able to duplicate this. 
```
 FAILED: F-Prime/Svc/Subtopologies/CdhCore/fpp-ast.json
  F-Prime/Svc/Subtopologies/CdhCore/fpp-analysis.json
  F-Prime/Svc/Subtopologies/CdhCore/fpp-loc-map.json
  PWD/build-fprime-automatic-va416x0-baremetal/sub-build-info-cache/F-Prime/Svc/Subtopologies/CdhCore/fpp-ast.json
  PWD/build-fprime-automatic-va416x0-baremetal/sub-build-info-cache/F-Prime/Svc/Subtopologies/CdhCore/fpp-analysis.json
  PWD/build-fprime-automatic-va416x0-baremetal/sub-build-info-cache/F-Prime/Svc/Subtopologies/CdhCore/fpp-loc-map.json


  cd
  PWD/build-fprime-automatic-va416x0-baremetal/sub-build-info-cache/F-Prime/Svc/Subtopologies/CdhCore
  && /usr/bin/python3
  PWD/lib/fprime/cmake/target/sub-build/../tools/arguments-from-file.py
  PWD/build-fprime-automatic-va416x0-baremetal/sub-build-info-cache/F-Prime/Svc/Subtopologies/CdhCore/fpp-cache/stdout.txt
  fpp-to-json
  PWD/lib/fprime/Svc/Subtopologies/CdhCore/CdhCore.fpp

  fpp-to-json

  PWD/lib/fprime/Svc/Subtopologies/CdhCore/CdhCore.fpp:47.30

      instance textLogger: Svc.PassiveTextLogger base id CdhCoreConfig.BASE_ID + 0x004000
                               ^

  error: undefined symbol PassiveTextLogger

  ninja: build stopped: subcommand failed.
```


## Context / Environment

Execute `fprime-util version-check` and share the output.

```
[root@dd01897f28a2 scythe]# fprime-util version-check
Operating System: Linux
CPU Architecture: x86_64
Platform: Linux-4.18.0-553.77.1.el8_10.x86_64-x86_64-with-glibc2.28
Python version: 3.9.20
CMake version: 3.26.0
Pip version: 25.3
Pip packages:
    fprime-tools==4.0.2a1
    fprime-gds==4.0.2a9
    fprime-fpp==3.1.0a10
Project submodules:
    https://github.com/nasa/fprime.git @ v4.0.1a1-83-g402c0317b
    https://github.com/fprime-community/fprime-baremetal @ b94a572
    https://github.com/fprime-community/fprime-vorago @ 5d0037a
```

## How to Reproduce

1. Disable text logging
2. Run fprime-utils generate  (see it fail)
3.

## Expected Behavior

`fprime-utils generate` should not fail when text logging is disabled

--- a/Svc/Subtopologies/CdhCore/CMakeLists.txt
+++ b/Svc/Subtopologies/CdhCore/CMakeLists.txt
@@ -1,3 +1,7 @@
+
+if (NOT FPRIME_ENABLE_TEXT_LOGGERS)
+    return()
+endif()
 add_fprime_subdirectory("${CMAKE_CURRENT_LIST_DIR}/CdhCoreConfig/")
 
 register_fprime_module(

